pages/index.vue

```diff
@@ -1,9 +1,79 @@
 <template>
-  <Tutorial/>
+
+  	<b-container>
+
+		<h1>Origami Annotation Tool</h1>
+		<b-row>
+		    <textarea :rows="fileTextLines" cols="300">
+				{{ this.fileText.join("\n") }}
+			</textarea>
+		</b-row>
+		<b-row>
+			<b-form>
+				<input type="file" @change="onFileSelected">
+			</b-form>
+		</b-row>
+
+	</b-container>
+
 </template>
 
 <script>
+
+import Vue from "vue";
+import Papa from "papaparse";
+
 export default {
-  name: 'IndexPage'
+  	
+	name: "IndexPage",
+
+	data() {
+		return {
+
+			fileInput: {},
+			fileText: ["File text here..."],
+			minimumTextLines: 2,
+			sourceList: []
+		}
+	},
+	
+	components: {
+
+	},
+
+	computed: {
+
+		fileTextLines() {
+
+			let numLines = this.fileText.length;
+			return numLines > 0 ? numLines : this.minimumTextLines;
+		}
+	},
+
+	methods: {
+
+		onFileSelected(p_event){
+
+			this.fileInput = p_event.target.files[0];
+			
+			// eslint-disable-next-line
+			// console.log(116, this.fileInput.name.split('.')[0]);
+
+			Papa.parse(this.fileInput, {
+				complete: results => {
+					this.sourceList = results.data[0];
+					console.log(results.data);
+					this.fileText = results.data;
+				},
+			});
+		}
+	},
+
+	// props: {
```

Comment:
Are those comments from the tutorial? I.e. Can we remove them?

Reply:
I'm remembering where and how to best use props. Just leftover commented out code. No props on the landing page thus far. Will cleanup in the next commit.